### PR TITLE
refactor: Use `ubuntu_lints` for linting

### PR DESCRIPTION
**Pull request by spydon**
This PR is still in draft since `ubuntu_lints` isn't published yet.

### CODE REVIEW

File: packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_model.dart [hunk 1]
```diff
@@ -101,8 +102,7 @@ class ManualStorageModel extends SafeChangeNotifier {
   }
 
   /// Notifies selection changes for auto-scrolling.
-  Stream get onSelectionChanged => _selectionController.stream;
-  final _selectionController = StreamController.broadcast();
+  ChangeNotifier selectionChangedNotifier = ChangeNotifier();
```
Comment (d-loose) [inline packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_model.dart:105]: Nice, that's much simpler!

File: packages/ubuntu_bootstrap/test/storage/manual/test_manual_storage.dart [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import 'package:collection/collection.dart';
+import 'package:flutter/cupertino.dart';
```
Comment (d-loose) [inline packages/ubuntu_bootstrap/test/storage/manual/test_manual_storage.dart:2]: Could we use `material.dart` or `widgets.dart` for consistency?

File: packages/ubuntu_bootstrap/pubspec.yaml [hunk 3]
```diff
@@ -46,7 +46,11 @@ dependencies:
   ubuntu_localizations: ^0.3.3
   ubuntu_logger: ^0.1.0
   ubuntu_provision: ^0.1.0
-  ubuntu_service: ^0.2.2
+  ubuntu_service:
+    git:
+      url: https://github.com/spydon/ubuntu-flutter-plugins.git
+      ref: ubuntu_service/param-generics
+      path: packages/ubuntu_service
```
Comment (d-loose) [inline packages/ubuntu_bootstrap/pubspec.yaml:53]: Let's publish 0.3.0 first and use it here :)

File: packages/ubuntu_bootstrap/test/storage/manual/test_manual_storage.dart (hunk 2 above)
Comment (spydon) [reply packages/ubuntu_bootstrap/test/storage/manual/test_manual_storage.dart:2]: Aah classic accidental auto-import, good catch!

File: packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_model.dart (hunk 1 above)
Comment (spydon) [reply packages/ubuntu_bootstrap/lib/pages/storage/manual/manual_storage_model.dart:105]: Yeah, not a big fan of emitting nulls over streams :sweat_smile:

File: packages/ubuntu_bootstrap/pubspec.yaml (hunk 3 above)
Comment (spydon) [reply packages/ubuntu_bootstrap/pubspec.yaml]: Updated to use the published version

**Review by d-loose — APPROVED** [review]
LGTM :+1:

**Pull request merged**